### PR TITLE
added merch object to javascript

### PR DESCRIPTION
**Pull request by aprilrochelle**
To Test:

Open main.js and check out the merch object for the tee, hat and vinyl

### CODE REVIEW

File: javascript/main.js [hunk 1]
```diff
@@ -4,4 +4,28 @@ console.log("hello");
...
+    link: "url"
+  },
+  {
+    item: "Semi-Solids Vinyl",
+    image: "/images/merch-vinyl.jpg",
+    price: 20,
+    description: "Keep the party going with the greatest Semi-Solids hits on VINYL! How cool are YOU?!",
+    link: "url"
+  }
+]
```
Comment (amillion3) [inline javascript/main.js]: add semicolon

**Comment by aprilrochelle** [comment]
Fixed

**Comment by amillion3** [comment]
👍

**Comment by ASlayton** [comment]
👍

**Comment by nathanpabst** [comment]
👍

**Pull request merged**